### ISSUE TITLE
More VC Type corrections

### ISSUE THREAD

**Issue by David-Chadwick** [w3c/vc-data-model#242]
In order for the examples to be consistent with the type text in section 5.1., we should not use the same values for the type of verifiable credential when different examples contain different properties  in the VC, nor should we use different types for credentials that contain the same properties. Consequently we should update the types in the following examples

Example 17,18, 19, 20, 21 change the type 'Credential' to 'VerifiableCredential' to be consistent with all the earlier examples.

Example 18. Change type to
```
  "type": ["VerifiableCredential", "ChildProofOfAgeCredential"],
```

**Comment by stonematt** [comment]
will the example context that's proposed in issue #254 address this?

**Comment by David-Chadwick** [comment]
I dont believe the context in #254 fixes this. But I note that 'Credential' has already been updated to 'VerifiableCredential' by someone else, so I have only changed the type in PR#274
